io/io/src/RRawFile.cxx

```diff
@@ -208,6 +211,41 @@ size_t ROOT::Internal::RRawFile::ReadAt(void *buffer, size_t nbytes, std::uint64
    return totalBytes;
 }
 
+void ROOT::Internal::RRawFile::PossiblyInjectFailure(void* buffer, size_t total_bytes, std::uint64_t offset)
+{
+   // check probability of error occurring here
+
+   // trigger particular failure
+   switch(GetFailureInjectionParams().failureType) {
+      case 0:
```

Comment:
Instead of integer literals, use enumerators introduced in `EFailureType`, e.g. `kBitFlip`.